docs/workers.md

```diff
@@ -333,6 +333,76 @@ when a user logs in on a new device and can be *very* resource intensive, so
 isolating these requests will stop them from interfering with other users ongoing
 syncs.
 
+Example `nginx` configuration snippet that handles the cases above. This is just an
+example and probably requires some changes according to your particular setup:
+
+```js
```

Comment:
```suggestion
```nginx
```

Reply:
Thanks! I didn't know there is a nginx specific formatter available!